### DISCUSSION TITLE
How many containers support by bottlerocket?

### DISCUSSION THREAD

**Discussion by rnz** [bottlerocket-os/bottlerocket#1399] [Q&A] [answered]
How many containers support by bottlerocket per host?

**Comment by jhaynes** [comment, marked as answer]
Hi @rnz thanks for the question. There is no upper limit imposed by the Bottlerocket OS per se. However, due to the way EKS does networking, there is a [max _pod_ count](https://github.com/bottlerocket-os/bottlerocket/blob/a290006674ac4fcb9c93434bf9fb4e29f09c64ff/packages/os/eni-max-pods) based on the instance type you select. For the ECS variant, similarly, there is no explicit cap. Instead, resource constraints based on the instance type and workload demands will determine how many containers can run on a given host.
